config/deployments/integration_test.toml

```diff
@@ -904,3 +904,8 @@ connector_list = "worldpayvantiv"
 [grpc_client.unified_connector_service]
 ucs_only_connectors = "paytm, phonepe"    # Comma-separated list of connectors that use UCS only
 ucs_psync_disabled_connectors = "cashtocode"    # Comma-separated list of connectors to disable UCS PSync call
+
+# Authentication to communicate between internal services using common_api_key, merchant id and profile id
+[internal_merchant_id_profile_id_auth]
+enabled = false # Enable or disable internal api key, merchant id and profile id based authentication
+internal_api_key = "test_internal_api_key" # API key for internal merchant id and profile id based authentication
```

Comment:
Let's remove this bit of config from the `config/deployments/integration_test.toml`, `config/deployments/sandbox.toml` and `config/deployments/production.toml` files. It would make more sense to maintain these privately, since we would have to specify some sort of a secret value for the API key when we enable this feature.